skbio/io/_exception.py

```diff
@@ -24,6 +24,11 @@ class UnrecognizedFormatError(FileFormatError):
     pass
 
 
+class GenbankFormatError(FileFormatError):
```

Comment:
I think this should be `GenBankFormatError` (GenBank instead of Genbank). This appears to be the canonical name.